src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -291,6 +291,8 @@ The `navigate` method takes these arguments:
 		- `'replace'`: the router will use `history.replaceState` to update the URL without adding a new entry into navigation.
 		- `'auto'` (default): the router will attempt `history.pushState`, but if the URL is not one that can be transitioned to, the current URL will remain with no changes to the browser history.
 
+For backward and forward navigation through the browser history you can simply combine `navigate` with the built-in `history.back`, `history.forward` &amp; `history.go` functions of the browser.
```

Comment:
```suggestion
For backward and forward navigation through the browser history, you can combine `navigate()` with the built-in `history.back()`, `history.forward()` and `history.go()` functions of the browser.
```
Quick notes that we spell out "and" in words, and we refrain from telling the reader what they should find "simple."  😄

Reply:
Fair :-)
Thanks for your advice, Sarah!